changes/ocd-3644.sql

```diff
@@ -0,0 +1,24 @@
+alter table openchpl.certification_criterion_attribute
+drop column if exists service_base_url_list;
+
+alter table openchpl.certification_criterion_attribute
+add column service_base_url_list bool not null default false;
+
+alter table openchpl.certification_result
+drop column if exists service_base_url_list cascade;
+
+alter table openchpl.certification_result
+add column service_base_url_list varchar(1024);
```

Comment:
Any reason to give it this restriction vs "text"? I haven't looked at the API code yet but make sure we check somewhere that the user-inputted value doesn't exceed this max length if you want to keep the size restriction in the DB. We have lots of maxlength properties and a reviewer that checks (only) some of them.

Reply:
All of our URLs have the same length requirement of 1024 characters. I'll check to see if this is in the maxlength validator in the Java